samples/KafkaFlow.Sample.ConsumerThrottling/KafkaFlow.Sample.ConsumerThrottling.csproj

```diff
@@ -17,7 +17,6 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\..\src\KafkaFlow.Admin\KafkaFlow.Admin.csproj" />
```

Comment:
Wouldn't the removal of Admin packages prevent access to the dashboard? Couldn't be a case of unreferenced at the build time but used in execution?

Reply:
https://github.com/Farfetch/kafkaflow/pull/486#discussion_r1422258927